src/Illuminate/Support/helpers.php

```diff
@@ -863,3 +863,17 @@ function with($object)
         return $object;
     }
 }
+
+if (! function_exists('loop')) {
```

Comment:
I initially added it to the foundation helpers just like `view()` but that wasn't acceptable as well, the options are:

1. Add a helper somewhere (foundation helpers, support helpers)
2. A method in the View facade `View::loop()`.
3. Unique shared variable in views `$__loop`.

What do you think? are there any other options?